### PR TITLE
repo2module: drop python-click dependency

### PR DESCRIPTION
**Pull request by FrostyX**
Other tools in this project use `argparse` so we would like to unify
this and drop the unnecessary dependency on `python-click`.

### CODE REVIEW

File: repo2module/tests/test_repo2module.py [hunk 1]
```diff
@@ -1,9 +1,6 @@
 import unittest
 import os.path
-
 import createrepo_c
-from click.testing import CliRunner
```
Comment (hroncok) [inline repo2module/tests/test_repo2module.py:5]: Nitpick: There is a (skipped) test that uses this class.

Comment (FrostyX) [reply repo2module/tests/test_repo2module.py:5]: True, thank you for pointing that out. I wasn't sure what to do with that call. I'll probably comment-out the code to have some example for when somebody decides to un-skip that test.

**Review by sgallagher — APPROVED** [review]
LGTM

**Comment by FrostyX** [comment]
Thank you for the review guys.

**Pull request merged**